blockchain/stake/staketx_test.go

```diff
@@ -1163,8 +1163,7 @@ func TestGetSStxNullOutputAmounts(t *testing.T) {
 		amtTicket)
 
 	// len commit to amts != len change amts
-	lenErrStr := "amounts was not equal in length " +
-		"to change amounts!"
+	const lenErrStr = "amounts was not equal in length to change amounts!"
```

Comment:
wow these error messages are gross.

besides that, it would be have some defined error variables to test this against (they don't need to be exported).

Reply:
Agreed.  I noticed they really need to be cleaned up and converted to `errors.Is` and all like the rest of the code, but was limiting the scope of this PR.  Plus, a lot of this module will eventually be rewritten/cleaned in favor of the `primitives` module.